### ISSUE TITLE
Error Occured after update to 1.10.5

### ISSUE THREAD

**Issue by masoodvahid** [hekmatinasser/verta#67]
```
Error
Object of class DateTime could not be converted to string 
```

and its refer to 
`vendor\hekmatinasser\verta\src\Verta.php:1457`

and
`\vendor\hekmatinasser\verta\src\Verta.php:234`

and
`\vendor\hekmatinasser\verta\src\Verta.php:363`

Verta in my code used liked this :
`$read_at = Verta::parse($record->read_at);`

> $record->read_at = 13990203

Laravel 7.0
Verta 1.10.5
PHP 7.4.7

**Comment by alireza2281** [comment]
the same happened for me

**Comment by bahaedini** [comment]
same for me with this command: `Verta::parse('1395-10-07 14:12:32');`

**Comment by hekmatinasser** [comment]
fix  error v1.10.6
